pkg/operator/apiserver/controller/workload/workload.go

```diff
@@ -354,6 +371,14 @@ func (c *Controller) updateOperatorStatus(ctx context.Context, previousStatus *o
 	return nil
 }
 
+func (c *Controller) addPrefix(name string) string {
```

Comment:
nit: please rename to `constructOperandNameFor(name string) string` or `constructOperandName(name string) string`